cmake/CmakeFunctionHelper.cmake

```diff
@@ -424,6 +429,41 @@ elseif(${TARGET} STREQUAL "ARM_GPU")
     "float"
   )
   foreach(data ${supported_types})
+  if(${MODEL} STREQUAL "RESNET_50")
+  add_gemm_configuration(
+    "${data}" 64 "false" "false" "false"
+    64 4 4 8 8 1 1 1 1 "no_local" "standard" "full" 4 "strided")
+  add_gemm_configuration(
+    "${data}" 32 "false" "false" "false"
+    64 8 4 4 8 1 1 1 1 "no_local" "standard" "partial" 4 "strided")
+  add_gemm_configuration(
+    "${data}" 32 "false" "false" "false"
+    64 4 8 8 4 1 1 1 1 "no_local" "standard" "partial" 4 "strided")
+  add_gemm_configuration(
+    "${data}" 64 "false" "false" "false"
+    64 4 4 8 8 1 1 1 1 "no_local" "standard" "partial" 4 "strided")
+    add_gemm_configuration(
+    "${data}" 32 "false" "false" "false"
+    64 8 4 4 8 1 1 1 1 "no_local" "standard" "partial" 1 "strided")
+    add_gemm_configuration(
+    "${data}" 32 "false" "false" "false"
+    64 8 4 4 8 1 1 1 1 "no_local" "standard" "partial" 2 "strided")
+    add_gemm_configuration(
+    "${data}" 16 "false" "false" "false"
+    64 4 4 4 4 1 1 1 1 "no_local" "standard" "partial" 4 "strided")
+    add_gemm_configuration(
+    "${data}" 16 "false" "false" "false"
+    64 4 4 4 4 1 1 1 1 "no_local" "standard" "partial" 1 "strided")
+  add_gemm_configuration(
+    "${data}" 64 "false" "false" "false"
+    64 4 4 8 8 1 1 1 1 "no_local" "standard" "partial" 2 "strided")
+  add_gemm_configuration(
+    "${data}" 128 "false" "false" "false"
+    64 4 8 16 8 1 1 1 1 "no_local" "standard" "partial" 4 "strided")
+  add_gemm_configuration(
+    "${data}" 16 "false" "false" "false"
+    64 4 4 4 4 1 1 1 1 "no_local" "standard" "partial" 2 "strided")
+  elseif(${MODEL} STREQUAL "VGG_16")
```

Comment:
So the existing configurations are considered to be VGG ones?

Reply:
Yes the configurations as they are right now on tip are optimised for VGG_16 as far as I'm aware.